### DISCUSSION TITLE
Will there be a new Jyers release with the latest Marlin 2.x out?

### DISCUSSION THREAD

**Discussion by ZeekWorld** [Jyers/Marlin#1973] [General Discussion]
I saw the latest Marlin 2.1 and was wondering if it will improve the latest Jyers build and if there is going to be an adaptation coming out soon for our Enders.
Thanks!

Zeek

**Comment by hatarez** [comment]
Same, not sure I should try this or wait for a 2.1 release. What do you guys (experts) think?
